modules/food-menu/fancy-menu.js

```diff
@@ -207,6 +207,8 @@ export class FancyMenu extends React.Component<Props, State> {
 				renderItem={this.renderItem}
 				renderSectionHeader={this.renderSectionHeader}
 				sections={(groupedMenuData: any)}
+				// remove this after react native fixes the sticky header positioning
+				stickySectionHeadersEnabled={false}
```

Comment:
Please include the relevant RN issue numbers in these comments

Reply:
I second this—can you include `see facebook/react-native#20956, facebook/react-native#21468, facebook/react-native#22444, facebook/react-native#20127` here?